packages/Uploader/src/components/File/File.js

```diff
@@ -71,25 +70,30 @@ function File(props) {
           <Popover isDark isEager>
             <Popover.Tip />
             <Popover.Trigger>
-              <Button.Icon
-                kind={Button.Icon.types.kind.MINOR}
-                onClick={() => {
-                  cancelFile(fileKey, onCancel);
-                }}
-                size={Button.Icon.types.size.SMALL}
-              >
-                <TimesIcon />
-              </Button.Icon>
+              {() => (
+                <>
+                  <Button.Icon
+                    aria-label={I18n.t("uploader.cancel_upload", { name })}
+                    kind={Button.Icon.types.kind.MINOR}
+                    onClick={() => {
+                      cancelFile(fileKey, onCancel);
+                    }}
+                    size={Button.Icon.types.size.SMALL}
+                  >
+                    <TimesIcon />
+                  </Button.Icon>
+                </>
+              )}
```

Comment:
No fragment needed here, but you do need to use the render prop arguments to make the tooltip work properly, like this:

```jsx
<Popover.Trigger>
  {(handler, a11yAttributes) => (
    <Button.Icon
      onMouseOver={handler}
      onMouseOut={handler}
      onFocus={handler}
      onBlur={handler}
      aria-label={I18n.t("uploader.cancel_upload", { name })}
      kind={Button.Icon.types.kind.MINOR}
      onClick={() => {
        cancelFile(fileKey, onCancel);
      }}
      size={Button.Icon.types.size.SMALL}
      {...a11yAttributes}
    >
      <TimesIcon />
    </Button.Icon>
  )}
</Popover.Trigger>
```

Reply:
This is _almost_ the same as the Storybook example:
https://github.com/acl-services/paprika/blob/master/packages/Popover/stories/examples/Tooltip.js